lib/draw.js

```diff
@@ -601,15 +601,28 @@ export default function (self) {
         m = self.style.scrollBarBoxMargin * 2;
       self.ctx.strokeStyle = self.style.scrollBarBorderColor;
       self.ctx.lineWidth = self.style.scrollBarBorderWidth;
-      en.horizontalBox.x =
-        rowHeaderCellWidth +
-        self.style.scrollBarBoxMargin +
-        (en.horizontalBar.width - self.scrollBox.scrollBoxWidth) *
-          (self.scrollBox.scrollLeft / self.scrollBox.scrollWidth);
+      if (self.frozenColumn > 0)
```

Comment:
Technically this _is_ just one statement (as with the `else` block), but because it's such a long one, maybe it's better to add braces `{ ... }` around these `if` and `else` blocks for clarity? Or did prettier format it like this?